cmd/parse.go

```diff
@@ -5,26 +5,58 @@ package cmd
 
 import (
 	"fmt"
+	"io"
+	"io/fs"
+	"os"
 
 	"seasonpackarr/internal/payload"
 	"seasonpackarr/internal/torrents"
+	"seasonpackarr/pkg/errors"
 
 	"github.com/spf13/cobra"
 )
 
-// parseCmd represents the test command
+// parseCmd represents the parse command
 var parseCmd = &cobra.Command{
-	Use:     "parse",
-	Short:   "Test the parse api endpoint for a specified release",
-	Example: `  seasonpackarr test parse --rls “Series.S01.1080p.WEB-DL.H.264-RlsGrp” --client "default" --host "127.0.0.1" --port 42069 --api "your-api-key"`,
+	Use:   "parse",
+	Short: "Test the parse api endpoint for a specified release",
+	Example: `  seasonpackarr test parse --rls “Series.S01.1080p.WEB-DL.H.264-RlsGrp” --client "default" --host "127.0.0.1" --port 42069 --api "your-api-key"
+  seasonpackarr test parse --rls “Series.S01.1080p.WEB-DL.H.264-RlsGrp” --torrent “Series.S01.1080p.WEB-DL.H.264-RlsGrp.torrent” --client "default" --host "127.0.0.1" --port 42069 --api "your-api-key"`,
 	Run: func(cmd *cobra.Command, args []string) {
-		torrentBytes, err := torrents.TorrentFromRls(rlsName, 5)
-		if err != nil {
-			fmt.Println(err.Error())
+		var torrentBytes []byte
+		var body io.Reader
+		var err error
+
+		if len(rlsName) == 0 {
+			fmt.Println("The release name can't be empty")
 			return
 		}
 
-		body, err := payload.CompileParsePayload(rlsName, torrentBytes, clientName)
+		if !cmd.Flags().Changed("torrent") {
+			torrentBytes, err = torrents.TorrentFromRls(rlsName, 5)
+			if err != nil {
+				fmt.Println(err.Error())
+				return
+			}
+		} else {
+			if len(torrentFile) == 0 {
+				fmt.Println("The path of the torrent file can't be empty")
+				return
+			}
+
+			if _, err := os.Stat(torrentFile); errors.Is(err, fs.ErrNotExist) {
+				fmt.Println("The specified torrent file doesn't exist", err.Error())
+				return
+			}
+
+			torrentBytes, err = os.ReadFile(torrentFile)
+			if err != nil {
+				fmt.Println(err.Error())
+				return
+			}
+		}
```

Comment:
Looks like it only reads either `--rls` or `--torrent`, so maybe it would make more sense to not use a flag here and instead use the first Arg and check if it contains the .torrent ext then read the file and if not it's a release title. 

You'd use it like `seasonpackarr test parse "Series.S01.1080p.WEB-DL.H.264-RlsGrp" --client "default" --host "127.0.0.1" --port 42069 --api "your-api-key"` and `seasonpackarr test parse "Series.S01.1080p.WEB-DL.H.264-RlsGrp.torrent" --client "default" --host "127.0.0.1" --port 42069 --api "your-api-key"`